### PR TITLE
skeleton of RatingsAndReviews component rendered with border colors

### PR DESCRIPTION
**Pull request by NylonEric**
* renamed a few components to follow RatingsAndReviews case style
* rendered basic <div> hierarchy of Ratings component
* basic styling for css grid and border colors
Screen Shot 2021-05-31 at 5 04 30 PM

### CODE REVIEW

File: client/src/components/QuestionsAndAnswers.jsx
Comment (baudot) [inline client/src/components/QuestionsAndAnswers.jsx:6]: Oopsie.

File: client/src/components/product_detail_page.jsx
Comment (baudot) [inline client/src/components/product_detail_page.jsx:4]: 👍

File: public/index.html
Comment (baudot) [inline public/index.html:5]: Typo'ed site name, should be "Atelier".

**Review by baudot — CHANGES_REQUESTED** [review]
Looks good! 
Just... there's a typo in the site name and that should get a quick-fix before approval.

File: public/index.html
Comment (NylonEric) [reply public/index.html:5]: Will do!

**Comment by NylonEric** [comment]
The title change has been implemented on the latest commit.

**Review by baudot — APPROVED** [review]
Thanks for the quick fix!

**Pull request merged**
